src/packet_distribution.cpp

```diff
@@ -333,6 +336,11 @@ void AOApplication::server_packet_received(AOPacket *p_packet)
     for (int n_element = 0; n_element < f_contents.size(); ++n_element) {
       QStringList sub_elements = f_contents.at(n_element).split("&");
 
+      sub_elements.replaceInStrings("#", "<num>")
+        .replaceInStrings("%", "<percent>")
+        .replaceInStrings("$", "<dollar>")
+        .replaceInStrings("&", "<and>");
+
```

Comment:
we have a function for this

Reply:
It was kinda tied to the AOPacket class. I made new commits that should handle this with static functions.